### PR TITLE
Typo and script fix

### PR DESCRIPTION
**Pull request by leboem**
Post checklist:

- [x] Ran local test?
- [x] Squashed commits?

I changed the git hook so that only changed files are checked.

### CODE REVIEW

File: _posts/2021-12-31-gs_00005_environment_variables.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ Hmm ... It is the same script as we used in the last post and there it worked.
 What went wrong? Can you guess? The answer is: we have overwritten the `PATH`
 variable so that it solely consists of your home directory. Since firefox is not
 located there, it cannot be found. By the way, also none of the Linux commands
-we've learned will not work anymore! Have a try:
```
Comment (kyaso) [inline _posts/2021-12-31-gs_00005_environment_variables.md:159]: I think the previous was grammatically correct, wasn't it?

**Review by kyaso — COMMENTED** [review]
Hook looks good.

One comment regarding typo in post.

File: _posts/2021-12-31-gs_00005_environment_variables.md (hunk 1 above)
Comment (leboem) [reply _posts/2021-12-31-gs_00005_environment_variables.md:159]: none ... wil not? I think it is either:
"also the Linux commands we've learned will not work anymore" or
"also none of the ... will work anymore" so just one negative part

Comment (kyaso) [reply _posts/2021-12-31-gs_00005_environment_variables.md:159]: Ohh, I missed the "none"!

**Review by kyaso — APPROVED** [review]
LGTM

**Pull request merged**